app/views/layouts/_feedback.html.slim

```diff
@@ -0,0 +1,56 @@
+- if params[:controller] == "results" || params[:controller] == "controlled_work_document_selections"
+  .gem-c-feedback__prompt
+    .govuk-width-container data-module="question-area"
+      span.gem-c-feedback__prompt-question-answer role="form"
+        h2.govuk-heading-s class="govuk-!-margin-bottom-0 govuk-!-margin-right-6"
+          = t("feedback.satisfaction.question")
+
+        = form_with(url: feedbacks_path(assessment_code: params[:assessment_code],
+                                        type: "satisfaction",
+                                        satisfied: "yes"),
+                                        local: false) do |form|
+          = form.govuk_submit "Yes", class: "gem-c-feedback__prompt-link-small-button", id: "yesButton"
+
+        = form_with(url: feedbacks_path(assessment_code: params[:assessment_code],
+                                        type: "satisfaction",
+                                        satisfied: "no"),
+                                        local: false) do |form|
+          = form.govuk_submit "No", class: "gem-c-feedback__prompt-link-small-button", id: "noButton"
+
+    .govuk-width-container data-module="link-area" hidden=true
+      span.gem-c-feedback__prompt-question-answer
+        h2.govuk-heading-s class="govuk-!-margin-bottom-0 govuk-!-margin-right-2"
+          = t("feedback.satisfaction.thank_you_message")
+        p.govuk-body-m class="govuk-!-margin-bottom-0 govuk-!-margin-right-6"
+          = t("feedback.satisfaction.feedback_link_html")
+- else
+  .gem-c-feedback__prompt data-module="freetext-feedback-area"
+    .govuk-width-container data-module="prompt-area"
+      .gem-c-feedback__prompt-content
+        .gem-c-feedback__prompt-questions--something-is-wrong
+          button.gem-c-feedback__prompt-link data-freetext-feedback="query"
+            = t("feedback.freetext.button")
+
+    .govuk-width-container data-module="freetext-area" hidden=true
+      .gem-c-feedback__form-heading
+        = t("feedback.freetext.heading")
+        p.govuk-body-m class="govuk-!-margin-bottom-4"
+          = t("feedback.freetext.heading_hint")
+          = form_with(url: feedbacks_path(assessment_code: params[:assessment_code],
+                                          type: "freetext",
+                                          page: @step),
+                                          local: false) do |form|
+            = form.govuk_text_area :text,
+                                    label: nil,
+                                    hint: nil,
```

Comment:
If we add `required: true` to this I think we can prevent the form from being submitted if no text is entered. What we need to do then is maybe _not_ hide this section if the field is blank (the browser should show a default indicator, which I think is fine).

Reply:
@HettieS another one to work on together, if possible.

Reply:
to chat about when pairing

Reply:
Updated the JS.